docs/user/security/securing-communications/index.asciidoc

```diff
@@ -26,65 +26,65 @@ NOTE: You do not need to enable the {es} {security-features} for this type of en
 For example:
 
 [source,sh]
 --------------------------------------------------------------------------------
-bin/elasticsearch-certutil csr -name kibana-server -dns some-website.com,www.some-website.com
+bin/elasticsearch-certutil cert -name kibana-server -dns localhost,127.0.0.1
 --------------------------------------------------------------------------------
 
-This will produce a ZIP archive named `kibana-server.zip`. Extract that archive to obtain the PEM-formatted CSR (`kibana-server.csr`) and
-unencrypted private key (`kibana-server.key`). In this example, the CSR has a common name (CN) of `kibana-server`, a SAN of
-`some-website.com`, and another SAN of `www.some-website.com`.
+This will produce a PKCS#12 file named `kibana-server.p12`, which contains the server certificate and private key.
 
-NOTE: You will need to use a certificate authority (CA) to sign your CSR to obtain your server certificate. This certificate's signature
-will be verified by web browsers that are configured to trust the CA.
+NOTE: In this example, the server certificate is signed by a locally-generated certificate authority (CA). This is not suitable for a
+production environment, and it will result in warnings in your web browser until you configure your browser to trust the certificate. Steps
+to configure certificate trust vary depending upon your browser and operating system. If you want to obtain a server certificate for a
+production environment, you can instead generate a certificate signing request (CSR) with `elasticsearch-certutil` using
+{ref}/certutil.html#certutil-csr[CSR mode].
 --
 
 . Configure {kib} to access the server certificate and private key.
 
-.. If your server certificate and private key are in PEM format:
+.. If your server certificate and private key are contained in a PKCS#12 file:
 +
 --
-Specify your server certificate and private key in `kibana.yml`:
+Specify your PKCS#12 file in `kibana.yml`:
 
 [source,yaml]
 --------------------------------------------------------------------------------
-server.ssl.certificate: "/path/to/kibana-server.crt"
-server.ssl.key: "/path/to/kibana-server.key"
+server.ssl.keystore.path: "/path/to/kibana-server.p12"
 --------------------------------------------------------------------------------
 
-If your private key is encrypted, add the decryption password to your <<secure-settings,{kib} keystore>>:
+If your PKCS#12 file is encrypted, add the decryption password to your <<secure-settings,{kib} keystore>>:
 
 [source,yaml]
 --------------------------------------------------------------------------------
-bin/kibana-keystore add server.ssl.keyPassphrase
+bin/kibana-keystore add server.ssl.keystore.password
 --------------------------------------------------------------------------------
+
+TIP: If your PKCS#12 file isn't protected with a password, depending on how it was generated, you may need to set
```

Comment:
Good question! I alluded to the difference [here](https://github.com/elastic/kibana/blob/master/packages/kbn-dev-utils/certs/README.md#pkcs-12), but didn't specify what elasticsearch-certutil itself does when you don't enter a password.

I was pretty sure that would result in a keystore with an empty password, but I wanted to test and make sure. By manually replacing the files in `packages/kbn-dev-utils/certs` and rerunning the unit tests in `src/core/server/utils/crypto/pkcs12.test.ts`, I verified that is indeed the case.

How do you feel about changing the tip to this?

> TIP: If you used `elasticsearch-certutil` to generate a PKCS#12 file and did not set a password, you need to set `server.ssl.keystore.password` to an empty string.

There are other ways this would be needed (such as using OpenSSL, as described in the link above) but I figure we should keep it concise here.

Reply:
Thanks for taking the time to verify this. I'm happy with the updated tip text